### PR TITLE
Image: Prevent image stretching while using Safari

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/1152

This is done by applying the https://github.com/siteorigin/so-widgets-bundle/issues/1141 fix in a media query that only targets IE 11.

### CODE REVIEW

**Review by Misplon — APPROVED** [review]

**Comment by Misplon** [comment]
Thanks. I can't test in IE 11 so I'll leave that you. I'm just waiting on a Safari user to confirm using a beta release before I merge.

**Comment by AlexGStapleton** [comment]
I can confirm this works in IE 11.

**Comment by Misplon** [comment]
No update from the user I sent the hotfix too, happy to go with what we have.

**Pull request merged**
